pkg/services/ngalert/notifier/channels/default_template.go

```diff
@@ -16,8 +16,12 @@ const (
 var DefaultTemplateString = `
 {{ define "__subject" }}[{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ if gt (.Alerts.Resolved | len) 0 }}, RESOLVED:{{ .Alerts.Resolved | len }}{{ end }}{{ end }}] {{ .GroupLabels.SortedPairs.Values | join " " }} {{ if gt (len .CommonLabels) (len .GroupLabels) }}({{ with .CommonLabels.Remove .GroupLabels.Names }}{{ .Values | join " " }}{{ end }}){{ end }}{{ end }}
 
+{{ define "__text_values_list" }}{{ $len := len .Values }}{{ if $len }}{{ $first := gt $len 1 }}{{ range $refID, $value := .Values -}}
```

Comment:
Does formatting of these lines affect the string output? If not I think we could add some whitespace and newlines to make this somewhat easier to read :)

Reply:
Not if we use `-` at the start or end of the braces, which removes whitespace. I did it at the end of each line, but wasn't sure if I should format it all like code considering most other templates are single line.

Reply:
Why not challenge the status quo – why are all the other ones on a single line? :D